tests/roundtrip.rs

```diff
@@ -0,0 +1,125 @@
+#[cfg(test)]
+mod tests {
```

Comment:
Why an extra inner module here?

Reply:
I initially did it because that's how the docs did it. I guess pointless at the present, but if we add assert-json-diff as a dev-dependency it could be useful.